package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "libvirt",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
0.10 has errors unfortunately but maybe I missed some nan macros, will look into it ....